allennlp/data/transforms.py

```diff
@@ -0,0 +1,466 @@
+from typing import Dict, Tuple, List, Iterable, Generic, TypeVar, Deque, Union
+import itertools
+from collections import deque, defaultdict
+
+
+import torch
+from torch.utils.data import Dataset as TorchDataset
+from torch.utils.data import IterableDataset as IterableTorchDataset
+
+
+from allennlp.common.registrable import Registrable
+from allennlp.data.instance import Instance
+from allennlp.data.vocabulary import Vocabulary
+
+from allennlp.data.fields import MetadataField
+
+from allennlp.common.util import lazy_groups_of
+
+
+Batched = Iterable[Instance]
+InstanceOrBatch = TypeVar("InstanceOrBatch", Iterable[Instance], Instance)
+
+
+class DatasetFromList(TorchDataset):
+    def __init__(self, instances: Iterable[InstanceOrBatch]):
+        self.instances
```

Comment:
self.instances = instances ?

Reply:
Good job this class is not used anywhere 😅

Reply:
please show TorchDataset some ♥️ 😂